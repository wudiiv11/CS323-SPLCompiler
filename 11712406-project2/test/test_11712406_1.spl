int foo(int a) {
    int b[3][3];
    int a[3][3];
    return 0;
}