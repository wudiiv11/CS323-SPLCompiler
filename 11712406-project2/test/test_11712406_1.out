Error type 3 at Line 2: redefine variable: a